src/core/src/main/java/org/apache/jmeter/report/processor/ExternalSampleSorter.java

```diff
@@ -91,9 +91,9 @@ public class ExternalSampleSorter extends AbstractSampleConsumer {
 
     private final BlockingQueue<Runnable> workQueue = new LinkedBlockingQueue<>();
 
-    private ThreadPoolExecutor pool;
+    private final ThreadPoolExecutor pool;
 
-    private volatile int nbProcessors;
+    private final int nbProcessors;
```

Comment:
Same thing: the field is private, and it is initialized only in constructor. So it can be safely made `final`.
It was IDEA that updated `volatile` -> `final` in `run inspection by name ...` -> `field may be final`